src/core/utils.ts

```diff
@@ -89,10 +89,15 @@ export function debuggerFactory(namespace: string) {
 
 const createUtilsDebugger = debuggerFactory('Utils')
 
-export function getAst(content: string): Program {
+export function getAst(content: string, isTsx?: boolean): Program {
+  const plugins = ['typescript', 'topLevelAwait']
```

Comment:
```suggestion
  const plugins: import('@babel/parser').ParserPlugin[] = ['typescript', 'topLevelAwait']
```
I think we can use `import` statement to import types dynamically since we depends `@babel/parser` indirectly.

Reply:
Yes, but we may need to track whether `Vue` will update the compiler plugin in the future.